static/app/components/replays/replayContext.tsx

```diff
@@ -29,6 +31,10 @@ type HighlightCallbacks = ReturnType<typeof useReplayHighlighting>;
 // It has state that, when changed, will not trigger a react render.
 // Instead only expose methods that wrap `Replayer` and manage state.
 interface ReplayPlayerContextProps extends HighlightCallbacks {
+  aiSummaryContext: {
```

Comment:
Let's start calling this replay summary instead of ai summary. Also `Context` is kind of redundant here, let's remove.

```suggestion
  replaySummary: {
```